zschema/leaves.py

```diff
@@ -246,16 +243,24 @@ class Enum(Leaf):
     INVALID = 23
     VALID = None
 
-    def __init__(self, values=[None,], *args, **kwargs):
```

Comment:
(I also cleaned up the mutable list which was used as a default argument in this initializer.)

Reply:
🎉